src/main/java/teammates/ui/webapi/FeedbackSessionClosingRemindersAction.java

```diff
@@ -13,16 +13,63 @@
 import teammates.common.util.EmailWrapper;
 import teammates.common.util.Logger;
 import teammates.common.util.RequestTracer;
+import teammates.storage.sqlentity.DeadlineExtension;
+import teammates.storage.sqlentity.FeedbackSession;
 
 /**
  * Cron job: schedules feedback session closing emails to be sent.
  */
-class FeedbackSessionClosingRemindersAction extends AdminOnlyAction {
+public class FeedbackSessionClosingRemindersAction extends AdminOnlyAction {
 
     private static final Logger log = Logger.getLogger();
 
     @Override
     public JsonResult execute() {
+        executeForDatastoreFeedbackSessions();
```

Comment:
Just want to ask: Are the `executeForDatastore...` functions removable after data migration is fully done?

Reply:
Yep, after migration can just easily remove the method definitions and calls